README.org

```diff
@@ -54,6 +54,14 @@ Test the parser:
   Letter { character: 'e', dots: Explicit([{Dot2, Dot1, Dot3}, {Dot1}]), constraints: EnumSet(Nofor) }
 #+END_SRC
 
+Build a release version:
+
+#+BEGIN_SRC shell
+  $ cargo build --release
+  $ cd ~/src
+  $ ln -s liblouis-[version] liblouis
```

Comment:
I get it that before testing the project needs to be built. But I do not understand why you want to create a symlink inside the src directory.